matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -179,6 +184,63 @@ var PhenoTips = (function (PhenoTips) {
         this._currentSortingOrder = "none";
     },
 
+    _initiateDisclaimers: function() {
+        $$('#checkbox-server-filters .mme-disclaimer-help').each(function(trigger) {
+            var disclaimerTextInput = trigger.up('label').down('.disclaimer');
+            if (disclaimerTextInput && disclaimerTextInput.value) {
+
+                var disclaimerContainer = new Element('span', {'class' : 'mme-disclaimer xTooltip hidden'});
+
+                var hideAllHelpOnOutsideClick = function (event) {
+                    if (!event.findElement('.xTooltip') && !event.findElement('.mme-disclaimer-help')) {
+                      disclaimerContainer.addClassName('hidden');
+                      document.stopObserving('click', hideAllHelpOnOutsideClick);
+                    }
+                }
+
+                var closeButton = new Element('span', {'class': 'hide-tool', 'title': 'Hide'}).update('×');
+                closeButton.observe('click', function(event) {
+                    if (event) {event.stop();}
+                    disclaimerContainer.addClassName('hidden');
+                    document.stopObserving('click', hideAllHelpOnOutsideClick);
+                });
+                disclaimerContainer.insert(closeButton);
+
+                var serverName = trigger.up('label').down('.serverName').value;
+                disclaimerContainer.insert(new Element('div', {'class' : 'server-name'}).insert(serverName));
+
+                var endTimeInput = trigger.up('label').down('.endTime');
+                if (endTimeInput && endTimeInput.value && new Date(endTimeInput.value)) {
+                    var time = new Date(endTimeInput.value);
+                    var lastRunInfo = this._MATCHES_LAST_RUN.replace('_TIME_', time.toISOString().split('T')[0]);
```

Comment:
This switches from local time zone to UTC. Matches computed at 10pm yesterday will be reported to have been computed today.